### PR TITLE
Remove ICRV information from edX course author and learner docs

### PR DESCRIPTION
**Pull request by srpearce**
## [DOC-3583](https://openedx.atlassian.net/browse/DOC-3583)

This PR removes in-course reverification from the edX course author and learner docs.

### Date Needed (optional)

7 March

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: @efischer19 
- [x] Doc team review (copy edit): @edx/doc
- [ ] Product review: @sstack22 

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

**Comment by sstack22** [comment]
I don't see any links to review, but if it's just removing that section and references of ICRV 👍

File: en_us/course_authors/source/course_features/credit_courses/credit_courses.rst [hunk 1]
```diff
@@ -7,25 +7,24 @@ Academic Course Credit
...
-   identity check involves several steps.
-
-#. :ref:`Add identity checks to the course <Add Identity Checks>`.
-
+#. :ref:`Add proctored exams to the course <CA_ProctoredExams>`. Proctored
+   exams are timed exams that learners take while being monitored by online
+   proctoring software. Only learners who have enrolled in the verified track
+   see the proctored exam option in the course. Learners who are not in the
+   verified track see the same exam in the course, but without the proctored
+   exam option.
```
Comment (efischer19) [inline en_us/course_authors/source/course_features/credit_courses/credit_courses.rst:27]: 👍 This looks way cleaner up here

**Review by efischer19 — APPROVED** [review]
Looks great!

**Comment by srpearce** [comment]
@catong or @grantgoodmanedX – can I get a review on this?

File: en_us/students/source/SFD_credit_courses.rst [hunk 2]
```diff
@@ -38,30 +37,22 @@ for the course.
...
 *****************************
-Complete Identity Checks
+Complete Proctored Exams
 *****************************
 
-When you take a course for academic credit, you have to complete a
-number of identity checks. Identity checks include periodic identity
-reverification inside the course and proctored exams.
+When you take a course for academic credit, you may have to complete proctored
+exams. Proctored exams are exams that use both time limits and online
```
Comment (catong) [inline en_us/students/source/SFD_credit_courses.rst]: Explaining that proctored exams "use online proctoring software" doesn't completely explain what it means to learners. 

I'd suggest something like "Proctored exams are exams that are monitored using online proctoring software and that have time limits."

**Comment by catong** [comment]
Just one comment then 👍

**Pull request merged**